modules/installation/pages/management-vm.adoc

```diff
@@ -2,68 +2,83 @@
 :toc:
 :toclevels: 3
 
-WARNING: This page is a work in progress. *This page will be automated, instead the user will be prompted before rebooting
-to disk or right after.*
+WARNING: This page is a work in progress. *This page will be automated, instead the user will be prompted before rebooting to disk or right after.*
 
-. Copy the management VM image from the Live CD into the appropriate location.
-+
-[source,bash]
-----
-mkdir -p /vms/images
-cp /data/*.qcow2 /vms/images/
-----
-. Copy the seed `meta-data.yml` and `user-data.yml` files into position
+. Copy the seed `meta-data` and `user-data` files into position
 +
 [source,code]
 ----
 mkdir -p /vms/cloud-init/management-vm
-cp /srv/cray/metal-former/libvirt/meta-data.yml /vms/cloud-init/management-vm
-cp /srv/cray/metal-former/libvirt/user-data.yml /vms/cloud-init/management-vm
+cp -p /srv/cray/management-vm/meta-data /vms/cloud-init/management-vm/
+cp -p /srv/cray/management-mv/user-data /vms/cloud-init/management-vm/
 ----
 . Generate an SSH key
 +
 [source,code]
 ----
 ssh-keygen
 ----
-. Use `yq` to update `user-data.yml`
+. Use `yq` to update `user-data`
 +
 [source,code]
 ----
-yq -i eval '(.users.[] | select(.name = "admin") | .ssh_authorized_keys) += "'"$(cat /root/.ssh/id_rsa.pub)"'"' /vms/cloud-init/management-vm/user-data.yml
+yq -i eval '(.users.[] | select(.name = "admin") | .ssh_authorized_keys) += "'"$(cat /root/.ssh/id_rsa.pub)"'"' /vms/cloud-init/management-vm/user-data
 ----
 . Create the cloud-init ISO
 +
 [source,bash]
 ----
-ln -snf user-data.yml /vms/cloud-init/management-vm/user-data
-ln -snf meta-data.yml /vms/cloud-init/management-vm/meta-data
 xorriso -as genisoimage \
     -output /vms/cloud-init/management-vm/cloud-init.iso \
     -volid CIDATA -joliet -rock -f \
     /vms/cloud-init/management-vm/user-data \
     /vms/cloud-init/management-vm/meta-data
 ----
-. Resize the VM image to at least 100G
+. Create the storage pool for the management VM.
++
+NOTE: By default `CAPACITY` is set to 100 (Gigabytes).
 +
 [source,bash]
 ----
+CAPACITY=100
 virsh pool-define-as management-pool dir --target /var/lib/libvirt/management-pool
 virsh pool-build management-pool
 virsh pool-start management-pool
 virsh pool-autostart management-pool
-virsh vol-create-as --pool management-pool --name management-vm.qcow2 --capacity 100G --format qcow2
-virsh vol-upload --pool management-pool management-vm.qcow2 /vms/images/management-vm-x86_64.qcow2
+virsh vol-create-as --pool management-pool --name management-vm.qcow2 --capacity ${CAPACITY}G --format qcow2
```

Comment:
I still like separation of pool creation and volume creation but it really doesn't matter.